findphotodates.py

```
@@ -702,21 +721,40 @@ def run_scan(directory, output, extensions, locate=False, quiet=False, debug=Fal
+                    location = geolocate(gps_lat, gps_lon)
+                    # Don't save coordinate strings as location (treat as None to allow retry)
+                    if location and is_coordinate_string(location):
+                        location = None
+                if date_taken:
+                    # Always preserve location if computed (for future --locate runs)
+                    # Use out_path (abspath) for output to preserve what user sees
+                    photo_data.append((out_path, date_taken, size_bytes, mtime_ns, gps_lat, gps_lon, location))
+                else:
+                    error_log.write(f"Warning: Could not extract date for '{out_path}'.\n")
+            except OSError as e:
+                error_log.write(f"Error: Could not access '{out_path}': {str(e)}\n")
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)